Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -15,7 +15,7 @@ public enum BTPayPalError: Error, CustomNSError, LocalizedError, Equatable {
     /// 3. HTTP POST request returned an error
     case httpPostRequestError([String: Any])
 
-    /// 4. The web approval URL, web redirect URL, or PayPal native app approval URL is invalid
+    /// 4. The web approval URL, web redirect URL, App Switch return URL, PayPal native app approval URL is invalid
```

Comment:
This addition can be removed now that we added an entirely new error case to the enum (`appSwitchReturnURLPathInvalid`)

```suggestion
    /// 4. The web approval URL, web redirect URL, PayPal native app approval URL is invalid
```